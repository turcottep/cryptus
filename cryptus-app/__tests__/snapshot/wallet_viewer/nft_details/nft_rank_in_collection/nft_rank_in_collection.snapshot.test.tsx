import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import NFTRankInCollection from "../../../../../components/profile/wallet_viewer/nft_details/nft_rank_in_collection/nft_rank_in_collection";

describe("<NFTRankInCollection />", () => {
  it("should display properly", async () => {
    const testPosition = 1234;
    const testTotal = 10000;
    const tree = renderer
      .create(<NFTRankInCollection position={testPosition} total={testTotal} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
