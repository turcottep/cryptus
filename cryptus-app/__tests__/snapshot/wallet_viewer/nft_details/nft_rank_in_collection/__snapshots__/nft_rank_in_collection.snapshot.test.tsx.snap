// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NFTRankInCollection /> should display properly 1`] = `
<div>
  rank: 1234/10000
</div>
`;
